src/types/models/package/layout.ts

```diff
@@ -0,0 +1,70 @@
+import { Nullable } from '../../utils';
+
+/** @public */
+export interface Image {
+  src: string;
+  link: Nullable<string>;
+  logoAltTextKey: Nullable<string>;
+}
+
+/** @public */
+export interface Style {
+  color: Nullable<string>;
+  backgroundColor: Nullable<string>;
+}
+
+/** @public */
+export interface TitleBarOptions {
+  title: boolean;
+  progressBar: boolean;
+}
+
+/** @public */
+export interface GlobalActionsOptions {
+  download: boolean;
+  saveAsLayout: boolean;
+  hideEvidenceSummary: boolean;
+  confirm: boolean;
+}
+
+/** @public */
+export interface HeaderOptions {
+  feedback: boolean;
+  breadcrumbs: boolean;
+  globalNavigation: boolean;
+  sessionBar: boolean;
+
+  globalActions: Nullable<GlobalActionsOptions>;
+  titleBar: Nullable<TitleBarOptions>;
+}
+
+/** @public */
+export interface BrandingBarOptions {
+  logo: Nullable<Image>;
```

Comment:
These properties still exist in the API, but changing them no longer does anything due to the New Signer Experience 😞 Probably still worth including here to be comprehensive?

Reply:
I'd say include them here for now